### PR TITLE
ref(api-docs): Remove unused API docs code

### PR DESCRIPTION
**Pull request by ceorourke**
Since the ecosystem team has migrated the API docs, these files, scenarios, and docs sections are no longer used and are therefore being removed.

### CODE REVIEW

**Review by markstory — APPROVED** [review]
Looks good to me. :clap:

File: src/sentry/api/endpoints/data_scrubbing_selector_suggestions.py [hunk 1]
```diff
@@ -4,25 +4,12 @@
 from rest_framework.response import Response
 
 from sentry import eventstore
-from sentry.api.base import DocSection
```
Comment (manuzope) [inline src/sentry/api/endpoints/data_scrubbing_selector_suggestions.py:7]: I think we can also remove the `DocSection` from base.py

**Review by manuzope — APPROVED** [review]
Tiny change. LGTM otherwise. Feel free to merge after deleting the DocSection enum.

**Comment by joshuarli** [comment]
simpsonsburnvillain

**Pull request merged**